app/src/organisms/Devices/emitPrepCommandsForModuleCalibration.ts

```diff
@@ -0,0 +1,133 @@
+import { CommandData } from '@opentrons/api-client'
+import { UseMutateAsyncFunction } from 'react-query'
+import {
+  HEATERSHAKER_MODULE_TYPE,
+  THERMOCYCLER_MODULE_TYPE,
+  TEMPERATURE_MODULE_TYPE,
+} from '@opentrons/shared-data'
+
+import type {
+  HeaterShakerDeactivateHeaterCreateCommand,
+  HeaterShakerDeactivateShakerCreateCommand,
+  HeaterShakerOpenLatchCreateCommand,
+  TCDeactivateBlockCreateCommand,
+  TCDeactivateLidCreateCommand,
+  TemperatureModuleDeactivateCreateCommand,
+  TCOpenLidCreateCommand,
+} from '@opentrons/shared-data'
+import type { CreateLiveCommandMutateParams } from '@opentrons/react-api-client/src/runs/useCreateLiveCommandMutation'
+import type { AttachedModule } from '../../redux/modules/types'
+
+type prepCommandForModuleCalibrationTypes =
```

Comment:
```suggestion
type PrepCommandForModuleCalibrationType =
```
The first letter should be capitalized to distinguish from variables.
In my understanding we avoid using plural for type.
For example,  in this case the type is one of the following types.

Reply:
oh ya, good catch thanks!